### ISSUE TITLE
generate_routing_topo_graph.sh bug

### ISSUE THREAD

**Issue by ScottDeng114514** [ApolloAuto/apollo#15221]
**Describe the bug**
Extra directory level in /apollo/scripts/generate_routing_topo_graph.sh
line 24:
`${APOLLO_BIN_PREFIX}/modules/routing/topo_creator/topo_creator \`
should be 
`${APOLLO_BIN_PREFIX}/modules/routing/topo_creator \`

**Comment by daohu527** [comment]
Yes, I also found this problem. I don’t know if it is caused by version adjustment. will fix later!

**Comment by ScottDeng114514** [comment]
> Yes, I also found this problem. I don’t know if it is caused by version adjustment. will fix later!
> 
> @ScottDeng114514 By the way, which Apollo version do you use 9.0?

9.x_alpha
